File: backend/src/main/java/com/cqupt/nepmserver/po/Statistics.java
package com.cqupt.nepmserver.po;
import lombok.Data;
import com.baomidou.mybatisplus.annotation.TableName;
import com.baomidou.mybatisplus.annotation.TableId;
import com.baomidou.mybatisplus.annotation.IdType;
import com.baomidou.mybatisplus.annotation.TableField;
@Data
@TableName("statistics")
public class Statistics {
    @TableId(value = "id", type = IdType.AUTO)
    private Integer id;
    private Integer provinceId;
    private Integer cityId;
    private String address;
    private Integer so2Value;
    private Integer so2Level;
    private Integer coValue;
    private Integer coLevel;
    private Integer spmValue;
    private Integer spmLevel;
    private Integer aqiId;
    private String confirmDate;
    private String confirmTime;
    private Integer gmId;
    private String fdId;
    private String information;
    private String remarks;

    //多对一
    @TableField(exist=false)
    private Supervisor supervisor;
    @TableField(exist=false)
    private GridProvince gridProvince;
    @TableField(exist=false)
    private GridCity gridCity;
    @TableField(exist=false)
    private Aqi aqi;
    @TableField(exist=false)
    private GridMember gridMember;


}

File: NEPG-网格员端/src/api/statistics.ts
import request from '@/utils/request'

export interface Statistics {
  provinceId?: number
  cityId?: number
  address?: string
  so2Value?: number
  so2Level?: number
  coValue?: number
  coLevel?: number
  spmValue?: number
  spmLevel?: number
  aqiId?: number
  confirmDate?: string
  confirmTime?: string
  gmId?: number
  fdId?: string
  information?: string
  remarks?: string
}

// 保存统计信息
export const saveStatistics = (data: Statistics) => {
  return request.post('/statistics/saveStatistics', data)
}
